### DISCUSSION TITLE
Not registered on airtel

### DISCUSSION THREAD

**Discussion by islam-mdshehab** [kyujin-cho/pixel-volte-patch#72] [Carrier-specific Experiences]
Followed your steps.. VOLTE showing in settings but alsp getting IMS not registered.. what should I do?
![Screenshot_20230227-035456](https://user-images.githubusercontent.com/126422222/221439823-077f1129-31f1-40c6-bd18-b31873907c73.png)

**Comment by samaltx** [comment]
I have same case with P6 in Malaysia. AIRTEL (Roaming) and Digi Networks. Both are IMS Unregistered and I'm not able to use VoLTE which was officially given by Google. Appreciate your help.
